components/skeleton/test/skeleton-group.visual-diff.js

```diff
@@ -0,0 +1,78 @@
+import puppeteer from 'puppeteer';
+import VisualDiff from '@brightspace-ui/visual-diff';
+
+describe('d2l-skeleton-group', () => {
+
+	const visualDiff = new VisualDiff('skeleton-group', import.meta.url);
+
+	let browser, page;
+
+	before(async() => {
+		browser = await puppeteer.launch();
+		page = await visualDiff.createPage(browser, { viewport: { width: 800, height: 3500 } });
+	});
+
+	after(async() => await browser.close());
+
+	before(async() => {
+		await page.goto(
+			`${visualDiff.getBaseUrl()}/components/skeleton/test/skeleton-group.visual-diff.html`,
+			{ waitUntil: ['networkidle0', 'load'] }
+		);
+		await page.bringToFront();
+	});
+
+	[
+		'all-skeleton',
+		'one-skeleton',
+		'no-skeleton',
+	].forEach((name) => {
+		it(name, async function() {
+			const rect = await visualDiff.getRect(page, `#${name}`);
+			await visualDiff.screenshotAndCompare(page, this.test.fullTitle(), { clip: rect });
+		});
+	});
+
+	[
+		{ name: 'make-one-skeleton', skeleton: true },
+		{ name: 'make-one-not-skeleton', skeleton: false },
+	].forEach((info) => {
+		it(info.name, async function() {
+			await page.$eval(`#${info.name}`, async(element, skeleton) => {
+				const panelGroup = element.querySelector('d2l-skeleton-group-test-wrapper').querySelector('d2l-collapsible-panel-group');
+				const panel = panelGroup.querySelector('d2l-collapsible-panel');
+				panel.skeleton = skeleton;
```

Comment:
You may want to `await panel.skeleton.updateComplete` here just to ensure that modifying the property has gone through Lit's update cycle.

Reply:
Is there benefit to awaiting the single property rather than the whole component? I'm used to doing stuff like this

```js
await panel.updateComplete
```

Reply:
Sorry my bad, that's what I meant -- there's no awaiting properties. :)